docs/Projects.md

```diff
@@ -8,30 +8,52 @@ title: "Tallwave Guidelines — Quality"
 # Quality
 
 # Readme's
+[README's](https://en.wikipedia.org/wiki/README) are the entry point to a project or code base. As such, they should be continuously updated as important information about the project and code base evolves and/or changes. In essence, a README is the instruction manual on how to get the content of a repository to do what it's intended to do.
+
+There are many examples of really good README's and even the [intro to GitHub](https://guides.github.com/activities/hello-world/) can be considered to be a README. There are also many examples of realy bad README's, including those that may be extensive but don't have the outcome originally intended.
+
+## Content Guidelines
 README's are there to give a description of the project, how to set it up, how to install it and how to get it to run.
-These are vital for onboarding new devs to the project as well as for future reference.
-Recommended content should be added to the following fields:
-## Title
-## Description
-## Tech Stack
-## Quick Start
- - forking, cloning & installing
- - running test scripts
- - starting the app
-
-## Plugins
-Please give a brief description of the plugins used (if possible) and any issues that are know to be problematic
-Links to the original repos &/or npm docs are useful
-
-## Project Structure
-Basic layout of folder structure and the 'what-goes-where' (api, client, React folder architecture etc.)
-
-## Database(s)
- - Description of the db's used (Relational/non-relational etc)
- - Set-up instructions
- - Tooling
- - GUI's that can/should be used
-
-## License
-
-## Contributors
+While the specifics of what should and should not be included in the document depend on the project or contents of the repository, the following informatioon should be imparted:
+
+##Recommended Structure
+The table of contents for a README should contain at least the following:
+> * **Title**
+> * **Description**
+> * **Tech Stack**
+> * **Quick Start**
+> * **Plugins**
+> * **Project Structure**
+> * **Database(s)**
+> * **License**
+> * **Contributors**
+
+**Title**: The title of the project/module/plugin is short but as decriptive and relevant as possible. 
+
+**Description**: More verbose introduction to the project and contents of the code.
+
+**Tech Stack**: A list of technologies used, for example:
+
+1. Node, 
+2. React, 
+3. Mongo, 
+4. AWS (or any external dependencies) etc.
+ 
+At a glance, the reader should get a general feel for what is needed to run the code.
```

Comment:
A more concise example would be calling out common tech stacks such as LAMP and MEAN.